app/main.py

```
@@ -192,11 +215,27 @@
 
 @app.post("/chat/{chat_id}/documents")
 async def chat_document_add(chat_id: str, document: Document, user_jwt: Annotated[str | None, Cookie()] = None):
-    return await wrap(chat_id, user_jwt, chat_document_add_success, document)
+    document = await wrap(chat_id, user_jwt, chat_document_add_success, document)
+    try:
+        sql_connection.cursor().execute(
+            "INSERT INTO `chat_users`.`statistics` (label, value) VALUES (?, 1) ON DUPLICATE KEY value = value +1;",
+            ['Documents']
+        )
+    except mariadb.Error as error:
+        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(mariadb.Error)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(mariadb.Error)`

(SIM105)